docs/v1/get-user-want-to-play-list.md

```diff
@@ -0,0 +1,65 @@
+<script setup>
+import SampleRequest from '../components/SampleRequest.vue';
+</script>
+
+# User Want to Play Lost
+
+A call to this endpoint will retrieve a giver user's want to play list, targeted by their username.
+
+[[toc]]
+
+## On-site Representation
+
+A user's want to play list can be found on on the user's Want to Play Games page:
+
+![Want to Play](/want-to-play-list.png)
+
+## HTTP Request
+
+<SampleRequest httpVerb="GET">https://retroachievements.org/API/API_GetUserWantToPlayList.php?u=MaxMilyin</SampleRequest>
+
+### Query Parameters
+
+| Name | Required? | Description                                                  |
+| :--- | :-------- | :----------------------------------------------------------- |
+| `z`  | Yes       | Your username.                                               |
+| `y`  | Yes       | Your web API key.                                            |
+| `u`  | Yes       | The target username.                                         |
+| `c`  |           | Count, number of records to return (default: 100, max: 500). |
+| `o`  |           | Offset, number of entries to skip (default: 0).              |
+
+## Client Library
+
+Not yet supported.
+
+:::
+
+## Response
+
+::: code-group
+
+```json [HTTP Response]
+{
+  "Count": 100,
+  "Total": 1287,
+  "Results": [
+    {
+      "GameID": 20246,
+      "Title": "~Hack~ Knuckles the Echidna in Sonic the Hedgehog",
+      "ImageIcon": "/Images/074560.png",
+      "ConsoleID": 1,
+      "PointsTotal": 1500,
```

Comment:
ConsoleName should be documented here too.

https://github.com/RetroAchievements/RAWeb/pull/2549/files#diff-64244a633e1006f4f355365ef4af32e102627504f2f3056e5a056a22afba744fR59